src/utils/deploymentInfo.js

```diff
@@ -14,6 +14,12 @@ const Environment = {
 const DomainName = {
   BIOMAGE: 'scp.biomage.net',
   BIOMAGE_STAGING: 'scp-staging.biomage.net',
+  HMS: 'cellenics.hms.harvard.edu',
```

Comment:
now that you use the account id, do you need these domain name harcodings?

Reply:
It is still being used to check whether to show the privacy policy or not, I think I could switch to using the accountId there too maybe, but I don't know if I should check on that in this PR?

Reply:
I'll remove the hms domainName from here